torch/distributed/rendezvous.py

```diff
@@ -193,7 +196,9 @@ def _error(msg):
 
     rank = int(query_dict["rank"])
     world_size = int(query_dict["world_size"])
-    use_libuv = query_dict.get("use_libuv", "0") == "1"
+    # libuv is the default backend for TCPStore. To enable the non-libuv backend,
+    # one needs to explicitly specify ``use_libuv=0`` in the URL parameter.
```

Comment:
I think we should support "USE_LIBUV" env var here as well (similar to env hander impl)